src/DetailsView/details-view-body.tsx

```diff
@@ -98,7 +98,11 @@ export class DetailsViewBody extends React.Component<DetailsViewBodyProps> {
             return null;
         }
 
-        return <DetailsViewLeftNav {...this.props} />;
+        const selectedPivot = this.props.visualizationStoreData
```

Comment:
I think you should be able to replace this with:

`this.props.visualizationStoreData?.selectedDetailsViewPivot`